src/util/fresh_symbol.cpp

```diff
@@ -11,6 +11,13 @@ Author: Chris Smowton, chris.smowton@diffblue.com
 
 #include "fresh_symbol.h"
 
+static size_t temporary_counter=0;
```

Comment:
Why do you need this outside get_fresh_aux_symbol? Can you add a comment on this? Also, I don't see the reset being called at all now..